src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/Debug/DebugProfileEnumValuesGenerator.cs

```diff
@@ -76,12 +76,16 @@ internal static ICollection<IEnumValue> GetEnumeratorEnumValues(ILaunchSettings
             var result = new Collection<IEnumValue>(
             (
                 from profile in profiles.Profiles
-                let value = new EnumValue { Name = profile.Name, DisplayName = profile.Name }
+                let value = new EnumValue { Name = profile.Name, DisplayName = EscapeMnemonics(profile.Name) }
                 select ((IEnumValue)new PageEnumValue(value))).ToList()
             );
 
             return result;
+        }
 
+        private static string EscapeMnemonics(string text)
+        {
+            return text?.Replace("&", "&&");
```

Comment:
Is this a problem in the admittedly edge case where someone already has `&&` (maybe if it represents some condition in code)?

Reply:
No I actually tested this, and works as you'd expect.